jobs/src/main/scala/dcos/metronome/jobrun/impl/JobRunExecutorActor.scala

```diff
@@ -41,6 +44,8 @@ class JobRunExecutorActor(
   val runSpecId = jobRun.id.toPathId
 
   override def preStart(): Unit = {
+    scheduleStartingDeadlineTimeout()
```

Comment:
personal preference... `scheduleTimeout(startingDeadline)`   I don't know why:) .  it is totally fine the way it is.

Reply:
yes, thank you. I like it more too...